R/polish_best_case_worst_case_emission_profile.R

```diff
@@ -0,0 +1,11 @@
+polish_best_case_worst_case_emission_profile <- function(data) {
+  data |>
+    select(-c(col_min_risk_category_per_company_benchmark(),
+              col_max_risk_category_per_company_benchmark(),
+              col_best_risk(),
+              col_worst_risk(),
+              col_count_best_case_products_per_company_benchmark(),
+              col_count_worst_case_products_per_company_benchmark())) |>
+    rename("amount_of_distinct_products" = col_n_distinct_products()) |>
```

Comment:
This "beautification" (rename) seems to belong to a more external "adapters" layer -- maybe move it into the catch-all `polish_emissions_any()`? -- which is as close to the output as possible.